Tasks/Buhayev/Task6/logic_user.py

```diff
@@ -0,0 +1,85 @@
+from db import Base
+
+class User():
```

Comment:
this is more like the traditional OOP, we can use the User class in many instances and create some descendants from it if needed 

but it has some logic-related methods as well, I would suggest to split it in two, one part for the user info storing, and another static part for the logic